src/main/scala/cromwell/logging/CromwellLogger.scala

```diff
@@ -22,8 +26,18 @@ class TerminalLayout extends LayoutBase[ILoggingEvent] {
      */
     val prefix = if (event.getMessage == "{}") s"[${TerminalUtil.highlight(129, "*")}] " else ""
 
-    s"$prefix[$level] $highlightedMessage\n"
+    s"$prefix[$level] $highlightedMessage\n${event.toStackTrace}"
+  }
+}
+
+object TerminalLayout {
+  val Converter = new ThrowableProxyConverter
+  Converter.start()
+
+  implicit class EnhancedILoggingEvent(val event: ILoggingEvent) extends AnyVal {
+    def toStackTrace: String = Converter.convert(event)
   }
 }
 
 class TerminalAppender extends ConsoleAppender
```

Comment:
Honestly I assumed it was part of the logback magic. Let me see what happens if I remove it.